README.rst

```diff
@@ -1,6 +1,6 @@
-*****
+=====
 Stone
-*****
+=====
```

Comment:
Another misguided attempt to troubleshoot the `docutils` update problem, but I _think_ this is more idiomatic. (See http://docutils.sourceforge.net/docs/user/rst/quickstart.html#document-title-subtitle.)